examples/dashboards/store.json

```diff
@@ -1677,7 +1677,7 @@
                      "step": 10
                   },
                   {
-                     "expr": "histogram_quantile(0.99, sum by (job, le) (rate(thanos_bucket_store_sent_chunk_size_bytes_bucket{job=~\"$job\"}[$__rate_interval])))",
+                     "expr": "histogram_quantile(0.50, sum by (job, le) (rate(thanos_bucket_store_sent_chunk_size_bytes_bucket{job=~\"$job\"}[$__rate_interval])))",
```

Comment:
Do we need to change this quantile?

Reply:
Yes, this is a typo fix. If you look below at the `legendFormat` it's supposed to be p50.

Reply:
Ack thanks! That makes sense!